vedo/plotter.py

```diff
@@ -4093,8 +4093,13 @@ def _default_keypress(self, iren, event) -> None:
             return
 
         elif key == "S":
-            vedo.file_io.screenshot("screenshot.png")
-            vedo.printc(r":camera: Saved rendering window to 'screenshot.png'", c="b")
+            def serial_fname(start=0):
+                fname = f"screenshot{int(start) if start > 0 else ''}.png"
+                return serial_fname(start + 1) if os.path.isfile(fname) else fname
+
+            fname = serial_fname()
+            vedo.file_io.screenshot(fname)
+            vedo.printc(rf":camera: Saved rendering window to {fname}", c="b")
```

Comment:
Alternative without recursion & without `screenshot0.png`
```suggestion
            fname = "screenshot.png"
            i = 1
            while os.path.isfile(fname):
                fname = f"screenshot{i}.png"
                i += 1
            vedo.file_io.screenshot(fname)
            vedo.printc(rf":camera: Saved rendering window to {fname}", c="b")
```